cnf-tests/README.md

```diff
@@ -69,12 +69,20 @@ There are two images used by the tests that can be changed using the following e
 # DPDK_TESTS_IMAGE
 ```
 
-For example, to change the `CNF_TESTS_IMAGE` run the following command
+For example, to change the `CNF_TESTS_IMAGE` with a custom registry run the following command
 
 ```bash
 docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig -e CNF_TESTS_IMAGE="custom-cnf-tests-image:latests" quay.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
 ```
 
+## Instruct the tests to consume those images from a custom registry
+
+This is done by setting the IMAGE_REGISTRY environment variable:
+
+```bash
+    docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig -e IMAGE_REGISTRY=my.local.registry:5000/ -e CNF_TESTS_IMAGE="custom-cnf-tests-image:latests" quay.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
```

Comment:
To be consistent can you change the image registry to this `IMAGE_REGISTRY="my.local.registry:5000/"`